packages/sx.js/src/utils/merkleVoting.ts

```diff
@@ -3,8 +3,8 @@ import { Contract } from '@ethersproject/contracts';
 
 import MerkleVotingABI from '../abis/MerkleVoting.json';
 
-const MERKLE_VOTING_RPC_PROVIDER = 'https://mainnet.skalenodes.com/v1/turbulent-unique-scheat';
-const MERKLE_VOTING_API = 'https://sx-api.razorscan.io';
+const MERKLE_VOTING_RPC_PROVIDER = 'https://mainnet.skalenodes.com/v1/elated-tan-skat';
+const MERKLE_VOTING_API = 'http://localhost:3000';
```

Comment:
this change needs to be reverted @rajkharvar

Reply:
yes, since tested by running the setup locally

Reply:
reverted in [77b16be](https://github.com/razor-network/sx-monorepo/pull/13/commits/77b16bee729ce7bc15ecee260ba42847c6f997c5)